session/pingpong/hermes_caller.go

```diff
@@ -288,16 +320,7 @@ func (ac *HermesCaller) GetConsumerData(chainID int64, id string) (HermesUserInf
 
 // GetProviderData gets provider data from hermes
 func (ac *HermesCaller) GetProviderData(chainID int64, id string) (HermesUserInfo, error) {
-	data, err := ac.getProviderData(chainID, id)
-	if err != nil {
-		return HermesUserInfo{}, err
-	}
-	err = data.LatestPromise.isValid(id)
-	if err != nil {
-		return HermesUserInfo{}, fmt.Errorf("could not check promise validity: %w", err)
-	}
-
-	return data, nil
+	return ac.getProviderData(chainID, id)
```

Comment:
This is wrong condition, because promise for provider signed by Hermes (operator)